src/pseudopeople/configuration/noise_configuration.py

```diff
@@ -23,7 +24,8 @@ def __init__(self, config: LayeredConfigTree):
         self._config = config
 
     def to_dict(self) -> dict:
-        config_dict: dict = self._config.to_dict()
+        # TODO: remove ignore when dropping support for Python 3.9
+        config_dict: dict = self._config.to_dict()  # type: ignore [assignment]
```

Comment:
Why are we ignoring this?

Reply:
This was actually a mistake - I was running mypy locally using an older version of LayeredConfigTree. Another example of the importance of keeping a fresh environment...